hearth/lib/hearth/middleware/retry.rb

```diff
@@ -73,20 +73,24 @@ def call(input, context)
           output.error = interceptor_error if interceptor_error
 
           if (error = output.error)
+            context.logger.debug('[Middleware::Retry] Request errored')
             error_info = @error_inspector_class.new(error, context.response)
             token = @retry_strategy.refresh_retry_token(token, error_info)
             break unless token
 
+            context.logger.debug('[Middleware::Retry] Sleeping before retry')
```

Comment:
worth adding the amount of time we plan to sleep?

Reply:
We could yeah